dts/bindings/sensor/invensense,mpu6050.yaml

```diff
@@ -16,3 +16,14 @@ properties:
       The INT signal default configuration is active-high.  The
       property value should ensure the flags properly describe the
       signal that is presented to the driver.
+
+  smplrt-div:
+    type: int
+    default: 7
```

Comment:
Need to explain in the description why the default value was selected
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#dt-bindings-default-rules